Resizetizer.NT/SharedImageInfo.cs

```diff
@@ -7,8 +7,20 @@ namespace Resizetizer
 {
 	internal class SharedImageInfo
 	{
+		public string Alias { get; set; }
+
 		public string Filename { get; set; }
 
+		public string OutputName =>
+			string.IsNullOrWhiteSpace(Alias)
+				? Path.GetFileNameWithoutExtension(Filename)
+				: Path.GetFileNameWithoutExtension(Alias);
+
+		public string OutputExtension =>
+			string.IsNullOrWhiteSpace(Alias) || !Path.HasExtension(Alias)
+				? Path.GetExtension(Filename)
+				: Path.GetExtension(Alias);
```

Comment:
Not 100% sure that we should use the alias extension... But seems most consistent. The user might make have to use a file that might have a bad extension, and this is the only way to work around it.